### PR TITLE
calculate type coverage for non-master default branch

### LINKED ISSUE #15: type coverage not being updated.
I have noticed recently that type coverage has stopped being updated for one of my projects ( https://github.com/azjezz/psl ), I'm not sure if this is an issue on my side, but the CI script has always been the same ( see: https://github.com/azjezz/psl/blob/develop/.travis.yml#L33 / https://github.com/azjezz/psl/blob/develop/composer.json#L42 ).

The latest CI build ran just a few seconds ago, but the latest update according to shepherd.dev is 4 months ago ( https://shepherd.dev/github/azjezz/psl ).

The only thing I can think of that has changed since then is the default branch name from `master` to `develop`

current type coverage: 100% ( 100.084 actually 😆 )
type coverage in shepherd: 91%

### PR DESCRIPTION
**Pull request by azjezz**
closes #15

### CODE REVIEW

File: src/GithubApi.php [hunk 1]
```diff
@@ -2,30 +2,53 @@
...
+    {
+        $config = Config::getInstance();
+        $github_token = Auth::getToken($repository);
+
+        $client = new Client(null, null, $config->gh_enterprise_url);
+        $client->authenticate($github_token, null, \Github\Client::AUTH_HTTP_TOKEN);
+
+        return $client;
+    }
 }
```
Comment (azjezz) [inline src/GithubApi.php:54]: this file was a mix of both tabs and spaces, I have switched it to spaces, but i can revert the changes if needed.

File: src/GithubData.php [hunk 2]
```diff
@@ -34,11 +34,9 @@ public static function storeMasterData(string $git_commit, array $payload) : voi
...
         );
 
-        self::setRepositoryForMasterCommit(
-            $git_commit,
-            $repository,
-            date('Y-m-d H:i:s', $payload['head']['date'] ?? date('U'))
-        );
+        /** @var string $date */
+        $date = date('Y-m-d H:i:s', $payload['head']['date'] ?? date('U'));
+        self::setRepositoryForMasterCommit($git_commit, $repository, $date);
```
Comment (azjezz) [inline src/GithubData.php:39]: this change was made here and in the `PsalmData` class to pass psalm type checking.

**Comment by azjezz** [comment]
there's a warning in GH actions 😛 

image

**Comment by muglug** [comment]
Thank you!

**Pull request merged**